download_cmd.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/facebookgo/stackerr"
+	"github.com/spf13/cobra"
+)
+
+type downloadCmd struct{}
+
+type releaseFiles struct {
+	UserFiles map[string]map[string]string `json:"userFiles"`
+	Checksums map[string]map[string]string `json:"checksums"`
+}
+
+type fileMap map[string][]byte
+
+func (d *downloadCmd) writeFiles(files fileMap) error {
+	for path, data := range files {
+		dir := filepath.Dir(path)
+		err := os.MkdirAll(dir, os.ModeDir|os.ModePerm)
```

Comment:
so you try to create a dir as well as a file for that path?
that seems unnecessary.